tools/uart-flash-server/ufserver.c

```diff
@@ -50,8 +50,8 @@
 #define CMD_HDR_ERASE 0x03
 #define CMD_ACK       0x06
 
-#define FIRMWARE_PARTITION_SIZE 0x4000
-#define SWAP_SIZE 0x4000
+#define FIRMWARE_PARTITION_SIZE 0x20000
+#define SWAP_SIZE 0x1000
```

Comment:
Lines 164-165 should be using `460800`, not `B460800`.

Reply:
Also noticed an old copy/paste for `-DWOLFSSL_DTLS` in a few tools Makefiles that should be removed.